src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_help.py

```diff
@@ -429,3 +429,15 @@
           text: >
             az acr build -r MyRegistry .
 """
+
+helps['acr import'] = """
+    type: command
+    short-summary: Imports to the container registry from source.
+    examples:
+        - name: Import an image to the target registry and inherits repository:tag from source.
+          text: >
+            az acr import -n MyRegistry --source SourceRegistry.azurecr.io/SourceRepository:SourceTag
```

Comment:
At least lower case repository name as it can only be lower case. I would recommend lower case login server and tag also. Tag is case sensitive.

Reply:
Lower-cased